kagami/database/database_service/admin_manger.py

```diff
@@ -0,0 +1,49 @@
+import bcrypt
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from ..models.admin import Admin, AdminPermissions
+
+
+class AdminService:
+    _session: AsyncSession
+
+    def __init__(self, session: AsyncSession) -> None:
+        self._session = session
+
+    def add_admin(self, name: str, raw_password: str, permissions: AdminPermissions):
+        bytes = raw_password.encode("utf-8")
+        salt = bcrypt.gensalt()
+        in_hash_password = bcrypt.hashpw(bytes, salt)
+
+        input = Admin(
```

Comment:
new_admin
if you use "input", this will override input function provided by python stdlib.